test/gui/widgets/test_changelog_widget.py

```diff
@@ -1,9 +1,17 @@
 from __future__ import annotations
 
+from typing import Any
+
 from randovania.gui.widgets.changelog_widget import ChangeLogWidget
 
 
 def test_create(skip_qtbot):
+    # mypy expects a QWidget (which they aren't) from these vars
+    qScroll0: Any  # QScrollArea
+    qScroll1: Any  # QScrollArea
+    qLabel0: Any  # QLabel
+    qLabel1: Any  # QLabel
```

Comment:
Also here, why are these not QWidgets? I would expect `currentWidget` to return one at least.

Reply:
In both cases the best thing to do is just `typing.cast`. `findChild` returns QWidget, but if you need a QLabel method that doesn't work.

Reply:
> Also here, why are these not QWidgets? I would expect currentWidget to return one at least.

It's possible that it can return a QWidget, but 9/10 that's not the case. In most cases it returns an object inheriting QWidget. Mypy raises an issue saying it has to be specifically QWidget

> In both cases the best thing to do is just typing.cast. findChild returns QWidget, but if you need a QLabel method that doesn't work.

Using `typing.cast` has worked, thanks for the tip